scripts/eachdist.py

```diff
@@ -466,28 +464,6 @@ def install_args(args):
             check=True,
         )
 
-    allfmt = "-e 'file://{}" if args.editable else "'file://{}"
-    # packages should provide an extra_requires that is named
-    # 'test', to denote test dependencies.
-    extras = []
-    if args.with_test_deps:
-        extras.append("test")
-    if extras:
-        allfmt += f"[{','.join(extras)}]"
-    # note the trailing single quote, to close the quote opened above.
-    allfmt += "'"
-
-    execute_args(
```

Comment:
Do we want to remove the call to `execute_args`? I think we just want to remove the [test] extras part.

Reply:
That makes sense, fixed it. Thank you